packages/mobile/src/home/__snapshots__/NotificationBox.test.tsx.snap

```diff
@@ -400,6 +400,7 @@ exports[`NotificationBox renders correctly for with all notifications 1`] = `
                       onResponderTerminate={[Function]}
                       onResponderTerminationRequest={[Function]}
                       onStartShouldSetResponder={[Function]}
+                      testID="undefined/backupKeyFlow6:getBackupKey/Button"
```

Comment:
Yeah so I needed to add a testID to notification buttons in order to properly test the "remind" button and to do that I needed to pass in the testID from the parent component as a prop to ensure uniqueness. Components elsewhere that aren't passing in a testID prop will show up with an "undefined" in the string (instead of not having a testID at all).

I could go back and pass through props from all parent components but that seemed out of scope.